infrastructure/libtor/Cargo.toml

```diff
@@ -7,16 +7,13 @@ license = "BSD-3-Clause"
 [dependencies]
 tari_common = { path = "../../common" }
 tari_p2p = { path = "../../base_layer/p2p" }
+tari_shutdown = { version = "^0.38", path = "../shutdown"}
 
 derivative = "2.2.0"
+libtor = "46.9.0"
```

Comment:
This might fail on windows

Reply:
It's an optional dependency in the base node/console wallet - so is the problem when `--all-features` is used?

Reply:
I added an explicit compile failure for windows builds that use `libtor` - although I'm not sure that is a great idea as windows may be supported one day

Reply:
@sdbondi https://github.com/delta1/tari/runs/8225310476?check_suite_focus=true#step:15:853

Reply:
Ah so the problem is that it's trying to build libtor on windows. Wonder how easy it would be to exclude that package from the build for windows e.g `cargo build --workspace --exclude tari_libtor`

Reply:
yep that's why I had put it under `[target.'cfg(unix)'.dependencies]` 

:shrug:

Reply:
nice hack :)